dali/ft/daftprogress.cpp

```diff
@@ -80,6 +80,8 @@ void DaftProgress::onProgress(unsigned __int64 sizeDone, unsigned __int64 totalS
         cycle_t elapsedTime = nowTime - startTime;
         cycle_t timeLeft = (cycle_t)(((double)elapsedTime) * (double)(totalSize - sizeDone) / (double)(sizeDone-startSize));
         unsigned __int64 msGone = cycle_to_nanosec(elapsedTime)/1000000;
+        if (0 == msGone)
+            msGone = 1;
```

Comment:
see comments below, because the code is confusingly using bytes / ms to approx kbpersec, this will cause a very large number of KBperSec to be seen when zero.
I think it should really calc. KB and divide by secs (see below)